sequence_date_range/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "sequence_date_range",
+    'description': """Module used to use the year of the date_to
+    into sequences (instead of date_from)""",
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```

Comment:
Please change this. See https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11